src/systemd/cockpit.service.in

```diff
@@ -14,11 +14,13 @@ DynamicUser=yes
 # otherwise systemd uses 'cockpit' even if it exists as a normal user account
 User=cockpit-systemd-service
 Group=cockpit-wsinstance-socket
-NoNewPrivileges=true
+NoNewPrivileges=yes
 ProtectSystem=strict
-ProtectHome=true
-PrivateTmp=true
-PrivateDevices=true
-ProtectKernelTunables=true
-RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
-MemoryDenyWriteExecute=true
+PrivateDevices=yes
+ProtectKernelTunables=yes
+MemoryDenyWriteExecute=yes
+
+# cockpit-tls speaks to the outside world via socket activation, and to cockpit-ws via Unix socket
+PrivateIPC=yes
+PrivateNetwork=yes
```

Comment:
I like that we deprive our network-facing component of the ability to interact with the network.  This feels really nice.